src/components/DownloadAndShareComponents/ShareLink/ShareLinks.tsx

```diff
@@ -56,6 +77,17 @@ const ShareLinks: FC<{
               }),
               () => {
                 setIsShareSuccessful(true);
+
+                props.lessonShared({
```

Comment:
Could this tracking event be on the LessonShare.page, with a onclick passed in to shareLinks ? I've refactored the other tracking events to be on the LessonShare.page in the other share PR and it would be good to keep them together if possible ?

https://github.com/oaknational/Oak-Web-Application/blob/7ad25a3ae8aecf7194f03d7b52f4354998b1f2b2/src/components/Lesson/LessonOverview/LessonOverview.page.tsx